asset/quickstart/list-assets/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START asset_quickstart_list_assets]
+
+// Sample asset-quickstart list assets.
```

Comment:
```suggestion
// Sample list-assets lists assets.
```

The default binary name is the name of the directory it's in (or the last part of the module name).

Reply:
Gotcha. Updated.